ds-caselaw-ingester/lambda_function.py

```diff
@@ -241,21 +241,32 @@ def store_file(file, folder, filename, s3_client: Session.client):
         print("Credentials not available")
 
 
-def send_new_judgment_notification(uri: str, metadata: dict) -> None:
+def send_new_judgment_notification(
+    source_uri: str, target_uri: str, metadata: dict
+) -> None:
+    tdr_metadata = metadata["parameters"]["TDR"]
+    if "/press-summary/" in source_uri:
+        doctype = "Press Summary"
```

Comment:
This is fine as is since we are still within the original ingester function here, but just wanted to flag that at this point we would have inserted the doc into the appropriate collection in ML and so _if we wanted to_, we could fetch the doc and determine the doc type from the collection.